gui/templates/home.html

```diff
@@ -527,10 +529,12 @@ <h2>Update Auto Rebalancer Settings</h2>
       <input id="variance" type="number" min="0" max="100" name="variance">
       <label title="The minutes we should wait after a failed attempt before trying again" for="wait_period">Wait Period (min): </label>
       <input id="wait_period" type="number" min="1" max="100" name="wait_period">
-      <label title="This enables or disables the Autopilot function which automatically acts upon suggestions on this page: /actions" for="autopilot">Autopilot: </label>
+      <label title="This enables or disables the Autopilot function which automatically acts upon suggestions on this page:" for="autopilot">Autopilot: </label>
```

Comment:
Intentional?

Reply:
I guess /actions was a typo which got copy pasted in other places too.

Reply:
https://github.com/cryptosharks131/lndg/pull/104/commits/ee94f33a80d62baf834036f02d0fe77943d48d31